general-concepts/licenses/text.xml

```diff
@@ -19,6 +39,86 @@ be used) then use the following syntax:
 LICENSE="|| ( foo bar )"
 </codesample>
 
+<subsection>
+<title>Determining the correct license</title>
+<body>
+
+<warning>
+Do not trust the license indicated by GitHub or other sites using
+automation to guess the license! The information presented is rarely
+complete, and may be incorrect.
+</warning>
+
+<p>
+To establish the correct value of <c>LICENSE</c>, you need to trace
+the licenses of all installed files. Normally, the licenses of output
+files (compiled executables, generated files) are the union of licenses
+of the respective input files.
+</p>
+
+<p>
+When looking for license information, the following should be
+considered:
+</p>
+
+<ol>
+	<li>
+		<c>COPYING*</c> and <c>LICENSE*</c> files distributed with
+		the package
+	</li>
+	<li>
+		explicit statements in documentation
+	</li>
+	<li>
+		explicit copyright notices in source and data files
+	</li>
+</ol>
+
+<p>
+The latter (more specific) options take precedence over the former.
+In particular, note that <c>COPYING*</c> files are frequently included
+as hardcopy of applicable licenses but the exact application of licenses
+and their versions is specified elsewhere.
+</p>
+
+<p>
+Please watch for license conflicts. If the license indicated
+by the package is incompatible with the licenses used by its sources
+(e.g. BSD/MIT package including GPL sources), please contact
+the licenses team for guidance. Do not add packages which seem
+to include license term violations.
+</p>
+
+</body>
+</subsection>
+
+<subsection>
+<title>GPL-x vs GPL-x+</title>
+<body>
+
+<p>
+It is a common mistake for developers to apply the incorrect variant
```

Comment:
We know that it is a common mistake, but could this be worded in a positive way nevertheless?